src/components/App.tsx

```diff
@@ -62,35 +86,41 @@ const App: FC<AppProps> = ({ client }) => {
     );
   };
 
-  return (
-    <FHIRClientProvider client={client}>
-      <PatientProvider>
-        <PathwayProvider
-          pathwayCtx={{
-            pathway: pathway,
-            setPathway: setPathwayCallback
-          }}
-        >
-          <div>
-            <Header logo={logo} />
-            <Navigation
-              service={service}
-              selectPathway={selectPathway}
-              setSelectPathway={setSelectPathway}
-            />
-          </div>
-          {selectPathway ? (
-            <PathwaysList
-              callback={setPathwayCallback}
-              service={service}
-              resources={patientRecords}
-            ></PathwaysList>
-          ) : (
-            <PatientView patientPathway={pathway} />
-          )}
-        </PathwayProvider>
-      </PatientProvider>
+  const renderPathwayProvider = (): ReactNode => {
+    return (
+      <PathwayProvider
+        pathwayCtx={{
+          pathway: pathway,
+          setPathway: setPathwayCallback
+        }}
+      >
+        <div>
+          <Header logo={logo} />
+          <Navigation
+            service={service}
+            selectPathway={selectPathway}
+            setSelectPathway={setSelectPathway}
+          />
+        </div>
+        {selectPathway ? (
+          <PathwaysList
+            callback={setPathwayCallback}
+            service={service}
+            resources={patientRecords}
+          ></PathwaysList>
+        ) : (
+          <PatientView patientPathway={pathway} />
+        )}
+      </PathwayProvider>
+    );
+  };
+
+  return !demo ? (
+    <FHIRClientProvider client={client as PathwaysClient}>
```

Comment:
I suspect the demo path will need some kind of FHIRClientProvider -- we're going to be referencing it downstream in the recommendation nodes via the accept/decline buttons, so if the client isn't present it might just crash. We don't want everything throughout the UI to be aware of whether it's in demo mode or not, but we also don't want to implement a whole static FHIR client that can stand in for the real thing.
Thoughts?

Reply:
For posterity - based on the discussion we just had offline, don't implement any mock FHIR client at this point since there's nothing that will actually exercise it. We can tackle that alongside the implementation of the real actions via the accept/decline button.